lib/xcode/install/install.rb

```diff
@@ -37,11 +37,14 @@ def validate!
         super
 
         help! 'A VERSION argument is required.' unless @version
+
+        fail Informative, "Version #{@version} doesn't exist." unless ENV.key?('XCODE_INSTALL_CACHE_DIR') || @url || @installer.exist?(@version)
```

Comment:
Why did you mandate the cache path (`ENV.key?('XCODE_INSTALL_CACHE_DIR')`) environment variable?